frontend/src/styles/NavBar.css

```diff
@@ -14,60 +15,92 @@
   }
 
   div.login-button {
+    opacity: 0.8;
+    position: relative;
+    width: 100px;
+    text-align:right;
     float: right;
-    padding: 5px;
-    color: #1DE28F;
-    font-weight: bold;
+    padding: 0px;
+    border: #202020 1px solid;
+    border-radius: 2px;
     cursor: pointer;
-    border: #1DE28F 2px solid;
-    border-radius: 10px;
+
+  }
+
+div.login-button img {
+    opacity: 0.8;
+    width: 100px;
+    bottom: 0;
+    display: inline-block;
   }
 
+
+div.login-button:hover{
+  border-color:#FFFFFF;
+}
+
   div.logout-button {
+    width:100;
+    height:30;
     float: right;
-    padding: 8px;
-    color: #1DE28F;
-    font-weight: bold;
+    padding: 4px;
     cursor: pointer;
-    border: #1DE28F 2px solid;
-    border-radius: 10px;
+    border: #202020 1px solid;
+    border-radius: 2px;
+  }
+  div.logout-button img {
+      position: right;
+      opacity: 0.8;
+      height: 30px;
+      width: 100px;
+      bottom: 0;
+      display: inline-block;
+    }
+  div.logout-button:hover{
+    border-color:#FFFFFF;
   }
 
   div.createMoment-button {
     display: inline-block;
     margin-left: auto;
     text-align: center;
-    padding: 8px;
+    padding-top: -1px;
+    padding-left: 20px;
```

Comment:
Too much space between the home and create moment buttons. Remove the padding-left.